assets/css/v2/pre_fare/simulation.scss

```diff
@@ -24,13 +24,40 @@
   .simulation {
     transform: scale(17.92%);
     transform-origin: top left;
+
+    --panel-width: 1080px;
+
+    &--duo {
+      width: calc(var(--panel-width) * 2);
+    }
+
+    &--left,
+    &--right,
+    &--solo {
+      width: var(--panel-width);
+    }
+
+    &--right > *,
+    &--solo > * {
+      margin-left: calc(var(--panel-width) * -1);
+    }
```

Comment:
Since all this is identical to the rules for `.screen-container` with only the selector being different, can we extract to a shared selector targeted at `.screen-container, .simulation`? This could be in a separate file, though I'm not sure what a good name would be. `pre_fare/viewport`?